interfaces/cython/cantera/ck2yaml.py

```diff
@@ -151,6 +154,7 @@ class InputError(Exception):
     the exceptional behavior.
     """
     def __init__(self, message, *args, **kwargs):
+        message += "\nPlease check {} for the correct Chemkin syntax.".format(url)
```

Comment:
Since `url` is now only used here, I think Bryan's suggestion was to just fold it into the message directly, that is:
```suggestion
        message += ("\nPlease check https://cantera.org/tutorials/ck2yaml-tutorial.html#debugging-common-errors-in-ck-files"
            "\nfor the correct Chemkin syntax.")
```
and delete the definition of `url` above.

Reply:
Thanks @speth for the revision. I think that I have pulled the latest changes from `main` branch and made the suggested change, but it seems like ruamel-yaml could not be installed successfully on the builds for all the systems with python3.8.  I am not sure what caused that, could you please give me some hints about how to fix it?